config/v1/types_infrastructure.go

```diff
@@ -573,6 +584,44 @@ type EquinixMetalPlatformStatus struct {
 	IngressIP string `json:"ingressIP,omitempty"`
 }
 
+// PowervsServiceEndpoint stores the configuration of a custom url to
+// override existing defaults of PowerVS Services.
+type PowerVSServiceEndpoint struct {
+	// name is the name of the Power VS service.
+	//
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:Pattern=`^[a-z0-9-]+$`
```

Comment:
this regex is not required anymore, the required marker will take care this.
```suggestion
```

Reply:
not true